### PR TITLE
Updated signature of build_application_zip

### PR DESCRIPTION
**Pull request by orangejenny**
For release once https://github.com/dimagi/commcare-hq/pull/24374/ has been deployed to all environments.

### CODE REVIEW

**Review by proteusvacuum — APPROVED** [review]
Will you handle merging when you think appropriate?

**Comment by orangejenny** [comment]
@proteusvacuum yep, I'll merge when it's safe, probably after June 3 since that's the next PNA deploy

**Comment by orangejenny** [comment]
Marking don't merge because this is throwing an error when downloading CCZs.

**Pull request merged**
